tabby-core/src/components/appRoot.component.pug

```diff
@@ -1,16 +1,20 @@
 title-bar(
   *ngIf='ready && !hostWindow.isFullscreen && config.store.appearance.frame == "full" && config.store.appearance.dock == "off"',
   (dblclick)='hostWindow.toggleMaximize()',
+  [class.hide-controls]='hostApp.platform !== Platform.Linux && !hostWindow.isFullscreen',
```

Comment:
should be controlled via an @Input on TitleBarComponent, not a class

Reply:
Done in the latest commit, window controls in the titlebar component is an ngIf now ✨